vl-kontrakt-behandlingsprosess-vedtak/v2/src/main/resources/xsd/beregningsgrunnlag/beregningsgrunnlag-svp-v2.xsd

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- edited with XMLSpy v2006 rel. 3 sp2 (http://www.altova.com) by XMLSPY 2006 Enterprise Ed. Release 3, Concurrent for 9 users (with SMP from 2006-06-21 to 2007-06-22) (Aetat Arbeidsdirektoratet) -->
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+			xmlns:bg="urn:no:nav:vedtak:felles:xml:vedtak:beregningsgrunnlag:svp:v2"
+			xmlns:be="urn:no:nav:vedtak:felles:xml:vedtak:beregningsgrunnlag:v2"
+			xmlns:felles="urn:no:nav:vedtak:felles:xml:felles:v2"
+			targetNamespace="urn:no:nav:vedtak:felles:xml:vedtak:beregningsgrunnlag:svp:v2"
+			elementFormDefault="qualified">
+	<xsd:import namespace="urn:no:nav:vedtak:felles:xml:felles:v2" schemaLocation="../felles-v2.xsd"/>
+	<xsd:import namespace="urn:no:nav:vedtak:felles:xml:vedtak:beregningsgrunnlag:v2" schemaLocation="beregningsgrunnlag-v2.xsd"/>
+
+	<xsd:complexType name="BeregningsgrunnlagSvangerskapspenger">
```

Comment:
Generelt, legg til følgende på complextypes. da blir det lettere å introdusere nye felter (attributter)
<xsd:anyAttribute processContents="lax" />

Reply:
Generelt 2, legg til følgende på complexTypes der det antas at det kan komme utvidelser (nye elementer) i fretiden. Da blir det lettere å legge elementer uten å gjøre en endring av namespace:
<xs:any namespace="##targetNamespace" processContents="strict" 
            minOccurs="0" maxOccurs="unbounded" />
      <xs:any namespace="##other" processContents="lax" minOccurs="0" 
            maxOccurs="unbounded" />

Se eksempelvis følgende for mer info:
https://docs.microsoft.com/en-us/previous-versions/aa468563%28v%3dmsdn.10%29#using-wildcards-to-create-open-content-models

Og ytterligere guidelines
https://docs.microsoft.com/en-us/previous-versions/aa468564(v=msdn.10)#the-guidelines

Reply:
Har lagt nå lagt inn "Generelt 1" gjennomgående. Når det gjelder "Genrelt 2"  har jeg litt vanskligheter med å avgjøre hvor det kan forventes å komme nye elementer i fremtiden.
Hele denne PR er egentlig en første versjon og vil bli gjennstand for flere runder der folk/team må bidra på de forskjellige områdene.